include/sddf/i2c/queue.h

```diff
@@ -25,18 +25,19 @@
 
 #define RESPONSE_ERR 0
 #define RESPONSE_ERR_TOKEN 1
-/* Start of payload bytes in response data */
+/* Start of payload bytes in response data (index of first non error byte that driver adds) */
 #define RESPONSE_DATA_OFFSET 2
 
 #define I2C_ERR_OK 0
 #define I2C_ERR_TIMEOUT 1
 #define I2C_ERR_NACK 2
 #define I2C_ERR_NOREAD 3
+#define I2C_ERR_OTHER 3 // can be used for driver specific implementations
 
 typedef uint8_t i2c_token_t;
 
 enum i2c_token {
-    /* END: Terminator for token list, has no meaning to hardware otherwise */
+    /* END: Terminator for token list, has no meaning to hardware otherwise (no need to use ever basically) */
```

Comment:
NOT TRUE. The end terminator is strictly needed!

Reply:
Not by the client when making requests though since the driver already pads empty space with 0x00.
Thats what I meant.